lgc/builder/ImageBuilder.cpp

```diff
@@ -465,7 +465,15 @@ Value *ImageBuilder::CreateImageLoad(Type *resultTy, unsigned dim, unsigned flag
     imageDescArgIndex = args.size();
     args.push_back(imageDesc);
     args.push_back(getInt32(tfe));
-    args.push_back(getInt32(((flags & ImageFlagCoherent) ? 1 : 0) | ((flags & ImageFlagVolatile) ? 2 : 0)));
+
+    // glc/slc/dlc bits
+    unsigned cachePolicyMask = (flags & ImageFlagVolatile) ? 2 : 0; // slc
```

Comment:
Could we add the magic numbers to builder.h (// Image operations)
enum {
  CachePolicyNone = 0,
  CachePolicyGlc =0x1,
  CachePolicySlc = 0x2,
  CachePolicyGlc = 0x4,
}

Reply:
Or to IntrinsDefs.h

Reply:
Good point, will add them.

Reply:
Yes, IntrinsDefs.h has a similar definition CoherentFlag though it is not used in builder (just used in patch).